### PR TITLE
Add Univalent GNU/Linux's logo

### PR DESCRIPTION
**Pull request by Jin-Asanami**

### CODE REVIEW

File: src/logo/builtin.c [hunk 1]
```diff
@@ -2000,6 +2000,35 @@ static const FFlogo* getLogoUbuntuSmall()
...
+        "UUUUUUUEEEEEE$2VVVVVVV$1      UUUUUUU\n"
+        "UUUUUUU       $2VVVVV$1       UUUUUUU\n"
+        " UUUUUUU       $2VVV$1       UUUUUUU\n"
+        "    UUUUUUU     $2V$1     UUUUUUU\n"
+        "       UUUUUUUUUUUUUUUUUUU\n"
+        "          UUUUUUUUUUUUU\n"
+    )
+    FF_LOGO_COLORS(
+        "34" //blue
+        "32" //green
```
Comment (LinusDierheimer) [inline src/logo/builtin.c:2025]: there is a `,` missing, this way the logo won't have any color.

File: src/logo/builtin.c [hunk 2]
```diff
@@ -2000,6 +2000,35 @@ static const FFlogo* getLogoUbuntuSmall()
...
+    )
+    FF_LOGO_COLOR_KEYS("32"); //blue
+    FF_LOGO_COLOR_TITLE("32"); //blue
```
Comment (LinusDierheimer) [inline src/logo/builtin.c:2028]: blue is 34, 32 is green. Which color do you actually want?

Comment (Jin-Asanami) [reply src/logo/builtin.c:2028]: Oops! Sorry, I forgot to correct it.
To be exact, it's 34, blue.

**Comment by LinusDierheimer** [comment]
There are build errors, please see the action tab and reopen

**Pull request merged**